test/run/test.utils.js

```diff
@@ -12,6 +12,10 @@ var sandbox = require('sandboxed-module');
 var binary = utils.normalizeBinary;
 
 sandbox.configure({
+  requires: {
+    // sandboxed-module is unable to find this module directly
```

Comment:
... because it is not listed at https://github.com/felixge/node-sandboxed-module/blob/v2.0.4/lib/builtin_modules.json and would therefore result in the module being resolved as a relative file path?

Reply:
I opened a PR upstream. This PR can be merged in the meanwhile because that package hasn't seen updates in 4 years.